circle.yml

```diff
@@ -9,8 +9,8 @@ machine:
 
 test:
   pre:
+    - wget -qO- https://cli-assets.heroku.com/install-ubuntu.sh | sh
```

Comment:
Lovely

Reply:
Haha

Reply:
Kind of intense that they have a `circle.yml` and `.travis.yml` both.